library/cc/headers_builder.cc

```diff
@@ -3,20 +3,20 @@
 namespace Envoy {
 namespace Platform {
 
-HeadersBuilder& HeadersBuilder::add(const std::string& name, const std::string& value) {
+HeadersBuilder& HeadersBuilder::add(std::string name, std::string value) {
   if (this->isRestrictedHeader(name)) {
     return *this;
   }
-  this->headers_[name].push_back(value);
+  this->headers_[std::move(name)].push_back(std::move(value));
   return *this;
 }
 
-HeadersBuilder& HeadersBuilder::set(const std::string& name,
-                                    const std::vector<std::string>& values) {
+HeadersBuilder& HeadersBuilder::set(std::string name,
+                                    std::vector<std::string> values) {
   if (this->isRestrictedHeader(name)) {
     return *this;
   }
-  this->headers_[name] = values;
+  this->headers_[std::move(name)] = values;
```

Comment:
std::move the `values`

Reply:
Oof! Good point. Sent out https://github.com/envoyproxy/envoy-mobile/pull/2694